governance.md

```diff
@@ -1,98 +1,137 @@
+## Governance Meetings and Decision-Making Process
 
-### Initial Charter
+Governance decisions, votes and questions should take place on the
+tekton-governance@googlegroups.com mailing list.
 
-This document will define how the committee is to manage the project until it has transitioned to an elected steering body, as well as what governance must be in place.
-The Kubernetes Steering Committee Charter Draft serves as a good example.
+TBD
 
-A charter should cover all of the following topics:
+## Elections
 
-* Scope of rights and responsibilities explicitly held by the committee
-* Committee structure that meets the requirements above
-* Election process, including:
-  * special elections in the case someone resigns or is impeached
-  * who is eligible to nominate candidates and how
-  * who is eligible to run as a candidate
-  * Voter registration and requirements
-  * election mechanics such as
-    * committee company representation quotas
-    * Limits on electioneering
-    * Responses to election fraud
-  * How are changes to the charter enacted, and by what process
-  * How are meetings conducted
-    * Recorded or not, and if not, how is the information shared
-    * How is work tracked? Example steering project board
-    * Is there a member note taker, or is there a neutral facilitator role that exists outside of the committee?
-    * Frequency, duration, and required consistency
-  * Committee decision-making process, and specifically those areas of action that require more/less consensus, e.g. modifications the charter
-  * Sub-Steering Committee governance structure (see this example)
+### Voter Eligibility
 
-## Transition Process
+Anyone who has at least 15 contributions in the last 12 months. Contributions
+include opening PRs, reviewing and commenting on PRs, opening and commenting on
+issues, writing design docs, commenting on design docs, helping people on slack,
+participating in working groups. The
+[dashboard on tekton.devstats.cd.foundation](https://tekton.devstats.cd.foundation/d/9/developer-activity-counts-by-repository-group-table?orgId=1&var-period_name=Last%20year&var-metric=contributions&var-repogroup_name=All&var-country_name=All)
+will show GitHub based contributions that are > 20 total.
+We expect to increase the contribution count required going forward.
 
-The transition process MUST:
+### Candidate Eligibility
 
-* Organize, execute, and validate an election for replacing bootstrap members (they may re-run, but would need to be re-elected in order to stay)
-* Define the term lengths for newly-elected individuals, ideally so not all members change out at once
-* Provide documentation for the community and committee members sufficient to smoothly continue the established practices of the committee
+Candidates themselves must be contributors who are eligible to vote, and must
+be nominated by contributors from at least two companies, which can include
+their own, and they can self-nominate.
```

Comment:
is it worth including the nomination process and the email address here as well? otherwise we'll have to dig it up again next time

Reply:
done